README.md

```diff
@@ -2,23 +2,39 @@
 
 A package for simplifying the EDA of different data types!
 
+## About
+
+**INSERT SUMMARY PARAGRAPH**
+
 ## Installation
 
 ```bash
 $ pip install pyxplor
 ```
 
-## Usage
+## Functions
+
+**INCLUDE BULLETED LIST OF FUNCTIONS (AND DATASETS IF APPLICABLE) THAT WILL BE INCLUDED IN THE FUNCTIONS (1-2 sentence description for each function/dataset)**
+
 
-- TODO
+## PyXplor Use in Python Ecosystem
+
+**INCLUDE A PARAGRAPH DESCRIBING WHERE YOUR PACKAGES FIT INTO THE PYTHON ECOSYSTEM (are there any other Python packages that have the same/similar functionality? Provide links to any that do. If none exist, then clearly state this as well)**
 
 ## Contributing
 
 Interested in contributing? Check out the contributing guidelines. Please note that this project is released with a Code of Conduct. By contributing to this project, you agree to abide by its terms.
 
+## Contributors
+
+- Ben Chen (@phchen5)
```

Comment:
I don't think the school's system uses Ben lol so let me just add my passport name.
```suggestion
- Po-Hsun (Ben) Chen (@phchen5)
```